docs/modules/ROOT/pages/integrations/logging.adoc

```diff
@@ -96,8 +96,12 @@ In this example the following happens:
 Logging URL generation in KubeArchive is controlled using a `ConfigMap` in
 the KubeArchive installation namespace.  URLs are generated using parameters
 that allow the URL to point to the specific log file associated with the
-given `Pod`. This `ConfigMap` requires a single entry, LOG_URL, whose string
-value will be interpolated using the other variables defined in the `ConfigMap`.
+given `Pod`. This `ConfigMap` requires two entries:
+
+. LOG_URL, whose string value will be interpolated using the other variables defined in the `ConfigMap`.
+. CONTAINER_NAME, whose value is a CEL expression that will be evaluated with data from the CloudEvent to provide the
```

Comment:
`CONTAINER_NAME` should not be required. If the sink needs the container name, it knows where to get it from.  It should not depend on the user providing it.

This is completely separate from what is contained in the logging `ConfigMap`.

Reply:
how does the sink know which log url is for which container?

Reply:
I reverted this change since KubeArchive will prove the `CONTAINER_NAME` CEL expression